### ISSUE TITLE
allow reordering of the now playing queue

### ISSUE THREAD

**Issue by ace518** [Mach5/supersonic#16]
The ability to reorder your now playing queue.  Perhaps a transparent dual arrow type display on the left you can grab to drag a song up and down in the list.

**Comment by timoreimann** [comment]
Good point -- I'd have found a way to reorder my playlist helpful too at times.

**Comment by ace518** [comment]
There's a lot of ideas i have for playlist management that could be helpful.  Once I get a chance to pull the code (and learn android development a bit) I want to get involved in this and contribute quite a bit.

**Comment by timoreimann** [comment]
I updated the issue topic to better reflect the desired functionality.

Glad to hear about another Android/Supersonic enthusiast. Looking forward to seeing more ideas and eventually code from you!
